core/sys/unix/pthread_netbsd.odin

```diff
@@ -0,0 +1,102 @@
+package unix
+
+import "core:c"
+
+pthread_t :: distinct u64
+
+SEM_T_SIZE :: 8
+
+PTHREAD_CONDATTR_T_SIZE    :: 16
+PTHREAD_MUTEXATTR_T_SIZE   :: 16
+PTHREAD_RWLOCKATTR_T_SIZE  :: 16
+PTHREAD_BARRIERATTR_T_SIZE :: 16
+
+PTHREAD_COND_T_SIZE    :: 40
+PTHREAD_MUTEX_T_SIZE   :: 48
+PTHREAD_RWLOCK_T_SIZE  :: 64
+PTHREAD_BARRIER_T_SIZE :: 48
+PTHREAD_ATTR_T_SIZE    :: 16
+
+pthread_cond_t :: struct #align(16) {
```

Comment:
Are you sure this is `#align(16)`, some other targets have also copied this from previous targets where this wasn't actually the case which caused problems.

Reply:
I will double check this. Thanks

Reply:
You are right. It should be 8.